share/rocm/cmake/ROCMConfig.cmake

```diff
@@ -1,6 +1,9 @@
 # ######################################################################################################################
-# Copyright (C) 2017 Advanced Micro Devices, Inc.
+# Copyright (C) 2023 Advanced Micro Devices, Inc.
 # ######################################################################################################################
 
-list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR})
-include(ROCMChecks)
+message(INFO
+    "Use of find_package(ROCM) is deprecated as of <version>, and will be removed in <version>, "
+    "please switch to find_package(ROCmCMakeBuildTools)."
```

Comment:
This should be edited for brevity. It's sufficient to say that it is deprecated and what to use instead.